movai_core_shared/core/message_client.py

```diff
@@ -142,13 +140,10 @@ def foraward_request(self, request_msg: dict) -> dict:
         """
         if "request" not in request_msg:
             request = {"request": request_msg}
-        self._zmq_client.send(request)
+        else:
```

Comment:
this is a bug, ,the request should be sent as {"request": {"req_type": ....}

Reply:
pushed a fix